pkg/endpoint/endpoint.go

```diff
@@ -1927,7 +1927,12 @@ func (e *Endpoint) ModifyIdentityLabels(source string, addLabels, delLabels labe
 		return err
```

Comment:
*(Not on a specific line of code on purpose.)*

I'd include more explanations in the commit description. Right now, your issue has a better explanation than the commit and I believe it would currently be hard for someone without context to understand your pull request. Reviewers are likely to struggle because of that.

One way to check this is to ask a colleague without context to review. If they fail to do so without reading the issue or asking you questions, then the commits probably don't have enough information.

Reply:
Thanks, I just updated the commit description. WDYT?